.travis.yml

```diff
@@ -60,7 +60,7 @@ matrix:
           popd
         - rustup component add clippy rustfmt
       script:
-        - cargo test --all
+        - cargo test --workspace --exclude e2e
```

Comment:
I would suggest adding a `default-members` that excludes the `e2e` crate to the `Cargo.toml` instead and then doing a `cargo test` (which will only run for the `default-members`).